ruby/read_json.rb

```diff
@@ -0,0 +1,19 @@
+"""Read JSON file from stdin line by line"""
```

Comment:
I'm just realizing that given how this is tested, we don't need to read the file line-by-line, we can read it all at once. 

The Python version of this reads line-by-line, too, and doesn't need to, either.

Reply:
And Ruby has a [`File::read` method inherited from `IO`](https://docs.ruby-lang.org/en/master/IO.html#method-c-read) that will read a file all at once and return a string

Reply:
Ok, rewriting to read file all at once. Is it ok to pass the file path as an arg then?